### PR TITLE
docs: add documentation for file endpoints

### PR DESCRIPTION
**Pull request by pyphilia**
ref #1528

### CODE REVIEW

File: src/services/item/plugins/file/schema.ts [hunk 1]
```diff
@@ -1,27 +1,45 @@
...
-      },
+  operationId: 'uploadFile',
+  tags: ['item', 'file'],
+  summary: 'Upload files',
+  description: 'Upload files to create corresponding items.',
+
+  querystring: customType.StrictObject({
+    id: Type.Optional(
+      customType.UUID({
+        description: 'Folder id in which the upload should be uploaded.',
```
Comment (spaenleh) [inline src/services/item/plugins/file/schema.ts:17]:
```suggestion
        description: 'Folder id in which the uploaded files should be created.',
```

File: src/services/item/plugins/file/schema.ts [hunk 2]
```diff
@@ -1,27 +1,45 @@
...
+      }),
     ),
-  ),
+    previousItemId: Type.Optional(customType.UUID()),
```
Comment (spaenleh) [inline src/services/item/plugins/file/schema.ts:20]: Add a description ?

**Review by spaenleh — CHANGES_REQUESTED** [review]
Maybe we need a bit more info ?

**Review by ReidyT — APPROVED** [review]
LGTM 🫕

**Review by spaenleh — APPROVED** [review]

**Pull request merged**